closed/src/java.base/share/classes/jdk/crypto/jniprovider/NativeCrypto.java

```diff
@@ -70,6 +70,8 @@ public static NativeCrypto getNativeCrypto() {
     }
 
     /* Native digest interfaces */
+    public static final native int loadCrypto();
```

Comment:
This should be package-private, not public.

Reply:
Done